staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -181,7 +184,7 @@ func (t *watcherBookmarkTimeBuckets) addWatcher(w *cacheWatcher) bool {
 	if !ok {
 		return false
 	}
-	bucketID := nextTime.Unix()
+	bucketID := int64(nextTime.Sub(t.createTime) / time.Second)
```

Comment:
Going back to my original comment about adding a test - can't you use FakeClock to test this?

Reply:
The essence of this problem is that the display clock changes, but the mononotonic clock does not change.
The FakeClock time can be changed using either of the following methods: SetTime and Step, both methods change FakeClock.time. The display clock and monotonic clock change simultaneously. Therefore, the problem cannot be simulated.

Reply:
OK - that sounds reasonable.

@liggitt - do you have any suggestions what else can we do here? Or can we go with as is?

Reply:
I also don't see a way to exercise the bug with the new code, so I'd be happy with this fix as is